src/runner/PlutoRunner.jl

```diff
@@ -1240,21 +1244,28 @@ function tree_data(@nospecialize(x::AbstractDict{<:Any,<:Any}), context::IOConte
     end
 end
 
-function tree_data_nt_row(pair::Tuple, context::IOContext)
+function tree_data_nt_row(@nospecialize(pair::Tuple), context::IOContext)
     # this is an entry of a NamedTuple, the first element of the Tuple is a Symbol, which we want to print as `x` instead of `:x`
     k, element = pair
     string(k), format_output_default(element, context)
 end
 
 
-function tree_data(@nospecialize(x::NamedTuple), context::IOContext)
+function tree_data(@nospecialize(x::NamedTuple), @nospecialize(context::IOContext))
```

Comment:
@fonsp Are multiple subtypes of `IOContexts` passed to Pluto during running? If yes, then this `@nospecialize` will probably help. Otherwise, it doesn't add much

Reply:
ah true, it's the same type every time. there was a point last year where we were just adding nospecialize randomly on everything. 

Do you have benchmarks for this function?

Reply:
> Do you have benchmarks for this function?

Benchmark:
```julia
julia --project -ie 'using Pluto; @time @eval Pluto.PlutoRunner.tree_data((; a=[1]), IOContext(IOBuffer()))'
```
tested on Julia 1.7.2.

## Before

```
  1.355317 seconds (5.00 M allocations: 269.918 MiB, 18.98% gc time, 99.59% compilation time)
```

## After

```
  0.924939 seconds (2.42 M allocations: 132.716 MiB, 21.76% gc time, 99.49% compilation time)
```